### DISCUSSION TITLE
Max Simultaneous calls

### DISCUSSION THREAD

**Discussion by Pcpro745** [DSheirer/sdrtrunk#1404] [Q&A] [answered]
Curious what the max simultaneous calls is? If i run 4 broadcastify feeds and have enough dongles will there be any dropped calls or will they queue up and upload?

**Comment by DSheirer** [comment, marked as answer]
The max is only constrained by your computer's resources.

In your Broadcastify Feed configuration, the 'Max Recording Age' controls how long a call audio is queued awaiting upload before it's thrown away.  So, if you try to push too much audio to the Feed, you'll risk loosing some audio due to age off.

**Reply by Pcpro745** [reply]
Oh ok thats good to know.
